src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -75,6 +75,7 @@ CLASS zcl_abapgit_object_fugr DEFINITION PUBLIC INHERITING FROM zcl_abapgit_obje
       IMPORTING
         !it_functions TYPE ty_function_tt
         !ii_log       TYPE REF TO zif_abapgit_log
+        !iv_version   TYPE uccheck
```

Comment:
Do we need this parameter? Function modules (as all sub other sub objects) don't specify their "own" ABAP language version. They have the same ABAP language version as their function group.